app/linguist.py

```diff
@@ -0,0 +1,49 @@
+import os.path
+import csv
+
+from .conllu_to_json_converter import convert
+from .train import train
+from jinja2 import Template
+from pathlib import Path
+
+def run(_dir, _code, _name):
+    output(_dir, 'stop_words.py', read_csv(_dir, _code, 'stop_words.csv'))
+    output(_dir, 'lemmatizer.py', read_csv(_dir, _code, 'lemmas.csv'))
+    output(_dir, 'norm_exceptions.py', read_csv(_dir, _code, 'same_spelling.csv'))
+    output(_dir, 'tokenizer_exceptions.py', read_csv(_dir, _code, 'contracted_words.csv'))
+    output(_dir, 'morph_rules.py', read_csv(_dir, _code, 'personal_pronouns.csv'))
+    output(_dir, '__init__.py', [_name, _code])
+    convert_ud(_dir, _code)
+    
+def read_csv(_dir, _code, filename):
+    result = []
+    try: 
+        with open(os.path.join(_dir, 'input', _code, filename)) as f:
+            reader = csv.reader(f, delimiter=',')
+            for row in reader:
+                stripped_row = list(map(str.strip, row))
+                result.append(stripped_row)
+    except:
+        result = None
```

Comment:
Would the return be `None` or an empty array in this case? If it can be an empty array, you can replace `result = None` to `pass`, because `results` already was declared on begin.